dev/movieDetails.jsx

```diff
@@ -2,13 +2,38 @@ import React, { Component } from 'react';
 import ReactDOM from "react-dom";
 import numeral  from "numeral";
 
-class MovieDetails extends Component{
-	componentWillMount() {
-	  document.addEventListener('click', this.handleClick, false);
+class MovieDetails extends Component {
+
+	constructor(props) {
+		super(props);
+		this.state = { data:''};
+		this.getmovieDetails = this.getmovieDetails.bind(this);
+		this.getmovieDetails(278927);
```

Comment:
This shouldn't be here. If what you want is to load a default movie you need to use the parameter you're sending from the parent component to this component. So in the `Search` constructor you need to set the state to the default value. I added a comment there too.